### DISCUSSION TITLE
在 dumi 约定书路由页面使用 pro layout，无法跳转到内容页面

### DISCUSSION THREAD

**Discussion by changhuaixin** [umijs/dumi#2136] [Q&A]
1. 在 dumi 创建的项目里创建一个约定路页面
2. 使用 pro layout，通过侧边栏跳转到设置 src/temp/mypage 后，无法路由跳转到 src/temp/mypage.tsx 文件作为页面内容。
![image](https://github.com/umijs/dumi/assets/61184708/7e556f4a-4525-4b62-84ce-0504587f579b)

3. mypage.tsx

export default () => 'welcome';

**Comment by PeachScript** [comment]
ProLayout 的使用问题应该提交到 pro-components 仓库
